frontend/src/features/admin-form/settings/components/AuthSettingsSection/AuthSettingsDisabledExplanationText.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box } from '@chakra-ui/react'
+
+import InlineMessage from '~components/InlineMessage'
+
+interface AuthSettingsDisabledExplanationTextProps {
+  isFormPublic: boolean
+  containsMyInfoFields: boolean
+}
+
+export const AuthSettingsDisabledExplanationText = ({
+  isFormPublic,
+  containsMyInfoFields,
+}: AuthSettingsDisabledExplanationTextProps) => {
+  return (
+    <Box my="2.5rem">
+      {containsMyInfoFields ? (
+        <InlineMessage mb="1rem">
+          To change Singpass authentication settings, close your form to new
+          responses and remove all existing Myinfo fields.
+        </InlineMessage>
+      ) : isFormPublic ? (
+        <InlineMessage mb="1rem">
+          To change Singpass authentication settings, close your form to new
+          responses.
+        </InlineMessage>
+      ) : null}
+    </Box>
```

Comment:
1. Looks like we might render an empty `<Box />` if form is not `containsMyInfoFields` and not `isFormPublic`.
2. Might be able to lift up the text over to a constants instead.

```suggestion
  const explanationText = containsMyInfoFields ? "To change Singpass authentication settings, close your form to new responses and remove all existing Myinfo fields." 
          : isFormPublic  
            ? "To change Singpass authentication settings, close your form to new responses."
            : null
  if (!explanationText) return null
 
  return (
    <Box my="2.5rem">
        <InlineMessage mb="1rem">
          {explanationText }
        </InlineMessage?
    </Box>
```

Reply:
resolved!